plugins/GSdx/GPUSettingsDlg.cpp

```diff
@@ -140,12 +140,12 @@ void GPUSettingsDlg::UpdateControls()
 		bool dx9 = i == 0;
 		bool dx11 = i == 1;
 		bool sw = i >= 0 && i <= 2;
-
-		ShowWindow(GetDlgItem(m_hWnd, IDC_LOGO9), dx9 ? SW_SHOW : SW_HIDE);
```

Comment:
I think you srew up the space. Empty line must be empty. And code must be aligned.